Dockerfile

```diff
@@ -34,4 +34,4 @@ VOLUME /var/log/conpot/
 
 EXPOSE 80 102 161/udp 502
 
-CMD ["/usr/local/bin/conpot", "--template", "default", "--logfile", "/var/log/conpot/conpot.log"]
+CMD ["/usr/local/bin/conpot", "--force", "--template", "default", "--logfile", "/var/log/conpot/conpot.log"]
```

Comment:
I'm not a big Docker honcho (OMG, I should really start to look into this) but: Does it make sense to deliver the script with the "--force" parameter by default?

Thing is - the --force parameter was added to let the user actively express his will to run a setting that we think should not be used in (except for demonstration purposes).

Now it looks like (pls correct me if I got this wrong) we're running the parameter as a default in a dockerized environment, which feels a little off :) any opinions?

Reply:
@creolis - 101% agreed, the -force option should imho not be used in such case.